lib/preprocessors/glimmer.js

```diff
@@ -14,6 +14,25 @@ function escapeRegExp(string) {
   return string.replace(/[$()*+.?[\\\]^{|}]/g, '\\$&'); // $& means the whole matched string
 }
 
+function arrayEq(arr1, arr2) {
+  for (const [idx, val1] of arr1.entries()) {
+    if (val1 !== arr2[idx]) {
+      return false;
+    }
+  }
+
+  return true;
+}
```

Comment:
though I see that in your specific usage, arr1.length is expected to be equal to arr2.length, slightly concerned that this assumption isn't always true and people in the future will misuse this function expecting it to contain the check for length equality

```suggestion
function arrayEq(arr1, arr2) {
  return arr1.length === arr2.length && arr1.every((val, idx) => val === arr2[idx]);
}
```